.github/workflows/python-publish.yml

```
@@ -27,5 +27,5 @@ jobs:
         TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
         TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
       run: |
-        python setup.py sdist
+        python setup.py sdist wheel
```

Think we want to use `bdist_wheel` based on [this doc]( https://wheel.readthedocs.io/en/stable/quickstart.html#quickstart ).

Would also suggest trying this locally to make sure it is behaving as expected 🙂

```suggestion
        python setup.py sdist bdist_wheel
```

---

@jakirkham good thing you reviewed -- I did test locally, but my change was not the same as what I used to test -- oops! :safety_vest: